### ISSUE TITLE
Suggestion: Shopping list - no items  categories

### ISSUE THREAD

**Issue by BluePixel4k** [estep248/Pocket-Plan#26]
Do not group items into categories.

**Comment by JakeGonder** [comment]
Interesting idea, we will see if more users request this feature, since it would be quite complicated to make it compatible with our current data structure

**Comment by k-svn** [comment]
Yes, please do away with the categories. Who categorizes 5 simple products before a purchase?

**Comment by jonathanengland** [comment]
I actually love the categories, but wish I could add my own custom ones. For instance, I have a board games wishlist but they're all under 'other' at the moment. I'm mentioning it here because disabling/custom categories seems a similar request (although they're polar opposites I know)

**Comment by Gertkeno** [comment]
I love the categories! Great for grocery shopping without doing laps around the store.

**Comment by pin88888** [comment]
I find good @jonathanengland suggestion. 
Maybe it is possible to solve the original request adding a blank (empty) category. 
Why is goof a List without categories? I.e.: I would use shopping lists also for books to be read or buy.
